template/Makefile.j2

```diff
@@ -39,8 +39,10 @@ version:
 	yq eval -i '.version = ${VERSION} | .appVersion = ${VERSION}' deploy/helm/{[ operator.product_string }]-operator/Chart.yaml
 
 config:
-	mkdir -p deploy/helm/zookeeper-operator/configs
-	cp -r deploy/config-spec/* deploy/helm/{[ operator.product_string }]-operator/configs
+	if [ -d "deploy/config-spec/" ]; then\
+		mkdir -p deploy/helm/zookeeper-operator/configs;\
```

Comment:
This seems like it was wrong before and is still wrong now :)
Didn't catch the hardcoded reference to ZK before so this can't really have worked....

Reply:
It worked since the error is in the portion that gets skipped :)
Wouldn't have worked for the other case, no.